strawberryfields/apps/graph/sample.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2019 Xanadu Quantum Technologies Inc.
+r"""
+Graph sampling
+==============
+
+**Module name:** :mod:`strawberryfields.future.apps.graph.sample`
+
+.. currentmodule:: strawberryfields.future.apps.graph.sample
+
+This module provides functionality for sampling of subgraphs from undirected graphs. The
+:func:`sample_subgraphs` function generates raw samples from
+:mod:`strawberryfields.future.apps.sample` and converts them to subgraphs using
+:func:`to_subgraphs`.
+
+Summary
+-------
+
+.. autosummary::
+    SAMPLE_DEFAULTS
+    sample_subgraphs
+    to_subgraphs
+
+Code details
+------------
+"""
+from typing import Optional
+import networkx as nx
+import numpy as np
+from .. import sample
+
+SAMPLE_DEFAULTS = {"distribution": "gbs", "postselect_ratio": 0.75}
+"""Dict[str, Any]: Dictionary to specify default parameters of options in :func:`sample_subgraphs`.
```

Comment:
As with the case of `Tuple` above, should it just be `dict` here?

Reply:
Depends. We tend to use `[x, y]` to mean "either x or y", whereas in `typing` syntax `Dict[x, y]` means dictionary mapping from x to y.